### DISCUSSION TITLE
Proposed changes to the Design System home page

### DISCUSSION THREAD

**Discussion by sshukor** [UKHomeOffice/design-system#452] [General]
Hi. I'm keen to get feedback on some proposed changes to the Home Office Design System home page. The changes (in red) are designed to make room for the addition of a new "Style guide" label to the top navigation bar. Also, I'm looking for views on when is the right time to move from the current 'Design System' hub to a broader 'Digital Service Manual' hub, of which the design system will be one element.

![Screenshot 2022-09-09 at 15 24 35](https://user-images.githubusercontent.com/101883702/189372834-9c94e6eb-78d6-4ac7-ac95-5e1680c874b9.png)

**Comment by tomyems** [comment]
That seems to make sense to me. I wonder whether 'Style guide' should be last on the right hand side. I wonder whether 'Contribute' would make more sense on the end?

The style guide could also have a box on the homepage if that would be more helpful - though we don't have analytics to know if anyone uses them.

I'm not sure about renaming - possibly, I don't have strong thoughts.

**Reply by sshukor** [reply]
Thanks Tom. Could 'Styles' and 'Style guide' on the same screen get confusing?

**Reply by tomyems** [reply]
Perhaps, have you considered 'Content style' to contrast with 'Design style'? Or maybe 'content' itself is jargon ...

**Reply by tomyems** [reply]
HMRC has this home page: https://design.tax.service.gov.uk/

**Reply by sshukor** [reply]
'Content style guide' could work.
